src/components/app/Risk/index.tsx

```diff
@@ -0,0 +1,103 @@
+/* eslint-disable react/display-name */
+import { useEffect, useState } from 'react';
+import { createStyles, makeStyles, Theme, Typography } from '@material-ui/core';
+import { useOnGet } from '@typesaurus/react';
+import { init as initFB } from '../../../utils/firebase';
+import { Grouping } from '../../../types/grouping';
+import _ from 'lodash';
+import {
+    amountToMMs,
+    getAverage,
+    getMedian,
+    getTvlImpact,
+} from '../../../utils/commonUtils';
+import { getStrategyTVLsPerProtocol } from '../../../utils/strategiesHelper';
+import { RiskChart } from '../../common/RiskChart';
+
+const useStyles = makeStyles((theme: Theme) =>
+    createStyles({
+        root: {
+            width: '100%',
+            padding: theme.spacing(2),
+            textAlign: 'center',
+        },
+    })
+);
+
+export const Risk = () => {
+    const [groups, setGroups] = useState<Grouping[]>([]);
+    const [items, setItems] = useState<any[]>([]);
+    const [isLoading, setIsLoading] = useState<boolean>(true);
+    const { groupings } = initFB();
+    const groupingId = 'default';
+    const [groupData, { loading, error }] = useOnGet(groupings, groupingId);
+    const classes = useStyles();
+    if (groups.length === 0 && groupData) {
```

Comment:
see here example: 
https://stackoverflow.com/questions/59070930/is-it-possible-to-use-a-custom-hook-inside-useeffect-in-react#:~:text=You%20can't%20use%20a,hook%20inside%20another%20custom%20hook.